apps/web/pages/settings/profile.tsx

```diff
@@ -156,6 +158,11 @@ function SettingsView(props: ComponentProps<typeof Settings> & { localeProp: str
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  function setTimePreferences(selectedTimeFormat: number) {
+    is24h(selectedTimeFormat === 24);
+    timeZone(dayjs.tz.guess());
```

Comment:
So this is setting the localStorage of the organizer, I don't think this is a bad idea but the profile page has a timeZone setting; if you can take that into account that'd be excellent.

Reply:
Hi @emrysal, can you kindly take a look again?